### DISCUSSION TITLE
I can not see the labels around my radar chart.

### DISCUSSION THREAD

**Discussion by manoj-singh-grokr** [recharts/recharts#3569] [Q&A]
I wanted to ask if there were any changes with recharts which moved the labels of PolarAngleAxis that get displayed around the radar chart to some other component. As of now, i can not see the labels around the chart. It was working till yesterday and today i can not see the labels at all.

![image](https://user-images.githubusercontent.com/100824766/236157856-48676f3b-a02d-42da-b019-2db7dc7022f5.png)

I looked at the radar chart in recharts website and the labels are not displayed there as well.

![image](https://user-images.githubusercontent.com/100824766/236158699-b3980073-361e-41ef-a77a-485e082df9ce.png)

In the image, subjects should be displayed around the chart but they're not.

**Comment by 201flaviosilva** [comment]
Hey,

Can you provide more information? Such as a piece of your code or a link to a sample project such as codesandbox/stackblitz/etc...

Here is a project example with labels working on a radar chart, with the newest version of recharts (2.5.0): https://stackblitz.com/edit/react-mqaptx?file=src%2FApp.js

**Comment by manoj-singh-grokr** [comment]
Hey Flavio,
Thank you for the quick reply. I figured out the problem.  I opened my app in a private tab. It was something to do with an extension I was using with my browser.
